Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+RUN useradd -m retdec
+WORKDIR /home/retdec
+ENV HOME /home/retdec
+
+RUN apt-get -y update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git bc graphviz upx cmake python zlib1g-dev flex bison libtinfo-dev autoconf pkg-config m4 libtool wget
+
+USER retdec
+RUN git clone --recursive https://github.com/avast-tl/retdec && \
+	cd retdec && \
+	mkdir build && \
+	cd build && \
+	cmake .. -DCMAKE_INSTALL_PREFIX=/home/retdec/retdec-install && \
```

Comment:
Shouldn't we install it into `/home/retdec/install`? The additional `retdec-` prefix seems redundant to me. Or is there a motivation behind the current path?

Reply:
The reason I use `retdec-install` is because if the end user adds more things to `/home/retdec/`, it can be a bit ambiguous what an `install/` folder is referring to.

Not opposed to doing it either way though, which do you want me to use? =)